src/clusterfuzz/_internal/datastore/data_handler.py

```diff
@@ -120,20 +120,48 @@ def get_testcase_by_id(testcase_id):
   return testcase
 
 
+@memoize.wrap(memoize.Memcache(MEMCACHE_TTL_IN_SECONDS))
+def _get_builtin_fuzzers():
+  return data_types.Fuzzer.query(ndb_utils.is_true(data_types.Fuzzer.builtin))
```

Comment:
We should just use this instead? https://github.com/google/clusterfuzz/blob/c2c875fa8378a732e88e1c155e12b7695672442f/src/clusterfuzz/_internal/fuzzing/__init__.py#L16